### ISSUE TITLE
Exception: Cannot set property 'services_only_admins' of undefined

### ISSUE THREAD

**Issue by jtatum** [slackhq/node-slack-client#23]
My bot based on the slack-client node module crashed a couple of times today. Looks like it got a message it couldn't handle. This is the mysterious backtrace. 

```
/app/node_modules/slack-client/src/client.js:546
        return this.team.prefs[message.name] = message.value;
TypeError: Cannot set property 'services_only_admins' of undefined
    at Client.onMessage (/app/node_modules/slack-client/src/client.js:546:46)
    at WebSocket.<anonymous> (/app/node_modules/slack-client/src/client.js:142:24)
    at WebSocket.emit (events.js:98:17)
    at Receiver.self._receiver.ontext (/app/node_modules/slack-client/node_modules/ws/lib/WebSocket.js:697:10)
    at Receiver.opcodes.1.finish (/app/node_modules/slack-client/node_modules/ws/lib/Receiver.js:397:14)
    at Receiver.expectHandler (/app/node_modules/slack-client/node_modules/ws/lib/Receiver.js:384:31)
    at Receiver.add (/app/node_modules/slack-client/node_modules/ws/lib/Receiver.js:93:24)
    at CleartextStream.firstHandler (/app/node_modules/slack-client/node_modules/ws/lib/WebSocket.js:678:22)
    at CleartextStream.emit (events.js:95:17)
    at CleartextStream.<anonymous> (_stream_readable.js:765:14)
```

**Comment by evansolomon** [comment]
This was fixed in 7e7dac795bda4c39215b4334be21436c0bf1b334 but there hasn't been a new release yet

**Comment by paulhammond** [comment]
1.4.0 was just released, fixing this bug.
